tests/integration/defs/accuracy/test_llm_api_autodeploy.py

```
@@ -596,6 +621,33 @@ def test_accuracy(self, model_id, world_size, enable_attention_dp,
 
         print_memory_usage("after evaluation")
 
+    @skip_pre_hopper
```

Flake8 (7.3.0)

[error] undefined name 'skip_pre_hopper'

(F821)